TimeTrackerClient/TimeTrackerClient/Presentation/HomeScreenViewModel.swift

```diff
@@ -1,19 +1,24 @@
 
 import Combine
+import Foundation
 
 public class HomeScreenViewModel: ObservableObject {
 
 	private let timeslotsLoader: TimeslotsLoader
 	private let userLoader: UserLoader
 	@Published public private(set) var timeslots: [TimeSlot] = []
+    let dateFormater = DateFormatter()
```

Comment:
asta vad ca nu e folosit, poti sa-l trimiti ca parametru la `sortDate` sau poti sa-l folosesti in View ca sa formatezi corect si atunci scoti formatarea din `sortDate`